docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Why is this a different granule than the block above? (Use this one for both)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>